refs/biblio.json

```diff
@@ -4645,6 +4645,16 @@
         "status": "Normative",
         "rawDate": "2015-03-01",
         "title": "OMG Unified Modeling Language"
+    },        
+    "UN/CEFACT": {
+        "authors": [
+            "United Nations Economic Commission for Europe"
+        ],
+        "href": "https://www.unece.org/cefact/codesfortrade/ccts_datatypecatalogue.html",
```

Comment:
Looking at this URL, it seems to link to 3 pdfs. Could you help us understand how you intend to cite this work in practice?

Reply:
I thought that whenever this source is linked the referenced version should come afterwards. But I could change the URL to the pdf with the most recent version.

Reply:
How about keeping this one as “latest”, but the adding UN-CEFACT-V1, UN-CEFACT-V2, UN-CEFACT-V3? Or would that be overkill? Alternatively, take a look at how other specs that have versions deal with this. Just a suggestion, as you might know best how these documents are to be cited.